.github/workflows/submitty_ci.yml

```diff
@@ -560,7 +560,7 @@ jobs:
         if: failure()
         with:
           name: cypress-screenshots
-          path: ${{env.SUBMITTY_REPOSITORY}}/site/cypress/screenshots
+          path: ${{env.SUBMITTY_REPOSITORY}}/site/cypress-image-diff-screenshots
```

Comment:
This should be both screenshots and image diff screenshots, if they exist

Reply:
done [944617e](https://github.com/Submitty/Submitty/pull/10260/commits/944617ef1f778debf671e2eb1aaf66c648ee9234)

Reply:
This should be BOTH cypress-screenshots AND the image diff screenshots, as the normal screenshots are for debugging failed tests